src/components/Dotting.tsx

```diff
@@ -515,8 +566,35 @@ const Dotting = forwardRef<DottingRef, DottingProps>(function Dotting(
       // tabIndex={0}
     >
       <canvas
-        ref={gotRef}
+        ref={gotBackgroundCanvasRef}
+        style={{
+          position: "absolute",
+          border: "1px solid #555555",
+          ...props.style,
+        }}
+      />
+      <canvas
+        ref={gotDataCanvasRef}
+        style={{
+          position: "absolute",
+          pointerEvents: "none",
```

Comment:
True! I previously tried to have only the first canvas to have all the styles but when we have a border style, due to the border, the canvas gets a little skewed since others do not have a border width 1px. Maybe you could change the structure later on? I'll create an issue regarding your concern.